hearth/sig/lib/hearth/configuration.rbs

```diff
@@ -2,6 +2,7 @@ module Hearth
   module Configuration
     def initialize: (**Hash[Symbol, untyped] options) -> void
 
-    def dup: () -> Hearth::Configuration
+    def merge: (Hash[Symbol, untyped] configuration) -> Configuration
+    def dup: () -> Configuration
```

Comment:
I don't think `dup` is a method anymore for this

Reply:
yeah, good call, we did remove that.

Related - I think the `-> Configuration` isn't actually quite correct here.  It returns an object of the class that includes this module.  I'm not quite sure how to correctly express that?